bindings/matrix-sdk-ffi/src/timeline/content.rs

```diff
@@ -44,6 +44,29 @@ impl TimelineItemContent {
                     source: Arc::new(MediaSource::from(content.source.clone())),
                 }
             }
+            Content::BeaconInfoState(beacon_state) => {
+                let Some(location) = beacon_state.last_location() else {
+                    return TimelineItemContentKind::FailedToParseMessageLike {
+                        event_type: "org.matrix.msc3672.beacon".to_string(),
+                        error: "Could not find beacon last location content".to_string(),
+                    };
+                };
```

Comment:
If the last location was not set, then I think the TimelineItemContent should not exist in the first place.